config/action.d/nftables-common.conf

```diff
@@ -11,84 +11,100 @@
 # used in all nftables based actions by default.
 #
 # The user can override the defaults in nftables-common.local
+# Example: redirect flow to honeypot
+#
+# [Init]
+# table_family = ip
+# chain_type = nat
+# chain_hook = prerouting
+# chain_priority = -50
+# blocktype = counter redirect to 2222
 
 [INCLUDES]
 
 after = nftables-common.local
 
 [Definition]
 
-# Option:  nftables_mode
-# Notes.:  additional expressions for nftables filter rule
-# Values:  nftables expressions
+# Option:  match
+# Notes.:  additional matches for nftables filter rule.
+#          leaving it empty will block all. (include udp and icmp)
+# Values:  nftables matches
 #
-nftables_mode = <protocol> dport \{ <port> \}
+match = <protocol> dport \{ <port> \}
 
 # Option:  actionstart
 # Notes.:  command executed once at the start of Fail2Ban.
 # Values:  CMD
 #
-actionstart = <nftables> add set <nftables_family> <nftables_table> <set_name> \{ type <nftables_type>\; \}
-              <nftables> insert rule <nftables_family> <nftables_table> <chain> %(nftables_mode)s <address_family> saddr @<set_name> <blocktype>
+actionstart = <nftables> add table <table_family> f2b-table
```

Comment:
Must check whether tables can get same name within different families.
At the moment it does not work anyway for `inet6` (because of wrong declaration of conditional init parameters, see comment below).